modules/express/src/clientRoutes.ts

```diff
@@ -616,10 +616,10 @@ export async function handleV2GenerateWallet(req: express.Request) {
   const bitgo = req.bitgo;
   const coin = bitgo.coin(req.params.coin);
   const result = await coin.wallets().generateWallet(req.body);
-  if (req.query.includeKeychains) {
-    return { ...result, wallet: result.wallet.toJSON() };
+  if (req.query.doNotIncludeKeychains) {
```

Comment:
Instead of changed the implementation customers may already be using, set `includeKeychains` to be checked or set to `true`.